README.md

```diff
@@ -117,33 +117,40 @@ The following prerequisites are required:
+* Keyrock is accessible via `localhost:3000`
+* CoatRack is accessible via `localhost:8080`
 
 Based on this, in order to make CoatRack work with the FIWARE Keyrock, you will need to:
 
-- First, pull and deploy the containers of keyrock via docker (as the Keyrock itself needs to be running in your local machine). To do this, just follow
-the tutorial in https://fiware-idm.readthedocs.io/en/latest/getting_started/index.html
+- At first you have to set-up/deploy your Keyrock instance. To do this, just follow the tutorial in https://fiware-idm.readthedocs.io/en/latest/getting_started/index.html
 - Please check that Keyrock is running correctly in your local machine by checking if you can access the Keyrock GUI,
 - If the Keyrock container is running and you can access the Keyrock GUI, then we need to start by registering CoatRack as application in Keyrock, to do this:
   - Create a new user account or log in with the account `admin@test.com`, which is created as the administrator account by default,
   ![KeyRock User Login](./spring-boot/admin/src/main/resources/static/images/keyrock/KeyRockLandingPage.png)
-  - After login, go to the application and click in the register button,
+  - After login, go to the application and click the register button,
   ![KeyRock register application](./spring-boot/admin/src/main/resources/static/images/keyrock/KeyRockRegisterApplication.png)
-  - Fill in all the fields in the registration form. The Callback URL must be `http:localhost:8080/login/aouth2/code/keyrock` as shown in the figure below. This is important because these values will be in the CoatRack configuration file later,
+  - Fill in all the fields in the registration form. The Callback URL must be `http://localhost:8080/login/oauth2/code/keyrock` as shown in the figure below. This is important because these values will be in the CoatRack configuration file later,
   ![KeyRock application form](./spring-boot/admin/src/main/resources/static/images/keyrock/KeyRockApplicationFormFilled.png)
-  - With your application already created (Attention: only the Callback URL needs to be the same, so for example you can define the application name as you wish), the next step will be to add the credentials that are required for CoatRack to access Keyrock to the CoatRack config in the `application-keyrock.yml`.
+  - In the step 2, you can upload your own image, to be shown as the application image,
+  ![KeyRock application form add avatar](./spring-boot/admin/src/main/resources/static/images/keyrock/KeyRockCreateApplicationAddAvatar.png)
+  - In the step 3, you can define the roles in your application, as seen in the image below,
+  ![KeyRock application form add roles](./spring-boot/admin/src/main/resources/static/images/keyrock/KeyRockCreateApplicationAddRoles.png)
+  - With your application already created, the next step will be to add the credentials that are required for CoatRack to access Keyrock to the CoatRack config in the `application-keyrock.yml`.
```

Comment:
```suggestion
  - After registering CoatRack as application in Keyrock, the next step will be to configure CoatRack so that it can access Keyrock.
```

Reply:
Done